google-cloud-spanner-hibernate-samples/spring-data-jpa-sample/pom.xml

```diff
@@ -75,6 +75,11 @@
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot-version}</version>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
Do you actually have to add this plugin? I thought it was built in.

Reply:
Ah good catch. Removed.